### ISSUE TITLE
Codespace does not see existing Azure subscription

### ISSUE THREAD

**Issue by coamtaker618** [MicrosoftDocs/vscodespaces#689]
TL;DR:
Codespaces says I do not have an Azure account despite the fact that Azure says I do.

Full Summary:
I setup trial Azure account to test out VS Codespaces but when I go to https://online.visualstudio.com/ to create my "Codespace plan" (which is going to start off as a self hosted machine, but I digress).  When I click the "Create Codespace" button it opens a tab that includes a dropdown for my Subscriptions---which is empty.  However when I click the provided link "Create a free Azure account" taking me to (https://azure.microsoft.com/en-us/free/ ) it says I already have an Azure account.  How do I get Visual Studio Codespaces to see the Azure account? They are on the same outlook.com email address & microsoft account.

Also concerning:
When I click on "Default Instance Type" All I see are "Basic,Standard,Premium" with no mention of self hosted!

**Comment by ryandle** [comment]
I'm seeing the same issue. I'm using Edge Chromium v 86.0.587.0. In the browser dev tools I can see that the request to login.microsoftonline.com was blocked:

![image](https://user-images.githubusercontent.com/578591/89236796-612f5200-d5a6-11ea-9887-b02f2a3b2405.png)

That causes getAuthToken() in create-plan-panel.tsx to throw (and the subscriptions dropdown becomes disabled):

![image](https://user-images.githubusercontent.com/578591/89236864-92a81d80-d5a6-11ea-93eb-e1768b2f78d4.png)

**Comment by apead** [comment]
I also have this problem.  I can't switch directory to access other subscriptions.

**Comment by jkeech** [comment]
Closing this as a duplicate of #39
